.circleci/config-2_1.yml

```diff
@@ -866,6 +898,23 @@ commands:
         path: /tmp/cassandra/build/test/logs
         destination: logs
 
+  run_cqlshlib_tests:
+    parameters:
+      no_output_timeout:
+        type: string
+        default: 15m
+    steps:
+    - run:
+        name: Run cqlshlib Unit Tests
+        command: |
+          export PATH=$JAVA_HOME/bin:$PATH
+          time mv ~/cassandra /tmp
+          cd /tmp/cassandra/pylib
+          ./cassandra-cqlsh-tests.sh ..
+        no_output_timeout: <<parameters.no_output_timeout>>
+    - store_test_results:
```

Comment:
can we also store the logs?  here is what we do for unit test

```
    - store_test_results:
        path: /tmp/cassandra/build/test/output/
    - store_artifacts:
        path: /tmp/cassandra/build/test/output
        destination: junitxml
    - store_artifacts:
        path: /tmp/cassandra/build/test/logs
        destination: logs
```

If a test fails, we need a way to to look at the results to debug

Reply:
That's 'not possible' atm unfortunately. Those tests are triggered through a sh script that builds a docker/virtual env and everything happens in there. Only the xml unit tests results file is fished out to be able to present test results. Logs and any other artifacts are lost _if_ they existed/are kept, which I haven't checked. We discussed sthg similar already [here](https://github.com/apache/cassandra/pull/747#discussion_r493906036).

We have the same problem in [jenkins](https://ci-cassandra.apache.org/job/Cassandra-trunk-cqlsh-tests/cython=no,jdk=jdk_1.8_latest,label=cassandra/392/console) as in circle where we only have the stdout to check any failures. 

Making that work in circle + jenkins + change the sh script etc is out of scope as the ticket is only about running them in circle were they weren't before. I would open a new ticket to improve these tests if we want to but I was not my intention to do so in this ticket.